eanalyze_staging.sql

```diff
@@ -0,0 +1,1308 @@
+  `updated_by_admin` varchar(45) DEFAULT NULL,
+  `updated_on_user` datetime DEFAULT NULL,
+  `updated_by_user` varchar(45) DEFAULT NULL,
+  `created_by_admin` varchar(45) DEFAULT NULL,
+  `created_on_user` datetime DEFAULT NULL,
+  `report_generated` varchar(100) DEFAULT NULL,
+  `last_new_shipment_mailed_on` datetime DEFAULT NULL,
+  `new_shipment_mail_count` int(11) NOT NULL DEFAULT 0,
+  `last_not_participated_mailed_on` datetime DEFAULT NULL,
+  `last_not_participated_mail_count` int(11) NOT NULL DEFAULT 0,
+  `qc_done` varchar(45) NOT NULL DEFAULT 'no',
+  `qc_date` date DEFAULT NULL,
+  `qc_done_by` varchar(255) DEFAULT NULL,
+  `qc_created_on` datetime DEFAULT NULL,
+  `mode_id` int(11) DEFAULT NULL,
+  `not_tested_reason` int(11) DEFAULT NULL,
+  `date_submitted` datetime DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1 COMMENT='Shipment for DTS Samples';
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `system_admin`
+--
+
+CREATE TABLE `system_admin` (
+  `admin_id` int(11) NOT NULL,
+  `first_name` varchar(255) DEFAULT NULL,
+  `last_name` varchar(255) DEFAULT NULL,
+  `primary_email` varchar(255) DEFAULT NULL,
+  `password` varchar(255) DEFAULT NULL,
+  `secondary_email` varchar(255) DEFAULT NULL,
+  `phone` varchar(255) DEFAULT NULL,
+  `force_password_reset` int(11) DEFAULT NULL,
+  `status` varchar(255) DEFAULT 'inactive',
+  `is_ptcc_coordinator` int(1) UNSIGNED NOT NULL DEFAULT 0,
+  `created_on` datetime DEFAULT NULL,
+  `created_by` varchar(255) DEFAULT NULL,
+  `updated_on` datetime DEFAULT NULL,
+  `updated_by` varchar(255) DEFAULT NULL,
+  `include_as_pecc_in_reports` int(1) NOT NULL DEFAULT 1
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `temp_mail`
+--
+
+CREATE TABLE `temp_mail` (
+  `temp_id` int(11) NOT NULL,
+  `message` text DEFAULT NULL,
+  `from_mail` varchar(255) DEFAULT NULL,
+  `to_email` varchar(255) NOT NULL,
+  `bcc` text DEFAULT NULL,
+  `cc` text DEFAULT NULL,
+  `subject` text DEFAULT NULL,
+  `from_full_name` varchar(255) DEFAULT NULL,
+  `status` varchar(255) NOT NULL DEFAULT 'pending'
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+-- --------------------------------------------------------
+
+--
+-- Structure for view `individual_final_report`
+--
+DROP TABLE IF EXISTS `individual_final_report`;
+
+CREATE ALGORITHM=UNDEFINED DEFINER=`root`@`localhost` SQL SECURITY DEFINER VIEW `individual_final_report`  AS  select `s`.`shipment_id` AS `shipment_id`,`s`.`shipment_code` AS `shipment_code`,`s`.`shipment_date` AS `shipment_date`,avg(ifnull(`spm`.`shipment_score`,0) + ifnull(`spm`.`documentation_score`,0)) AS `mean_shipment_score` from (`shipment` `s` join `shipment_participant_map` `spm` on(`s`.`shipment_id` = `spm`.`shipment_id`)) where `s`.`is_official` = 1 and `s`.`shipment_id` <= 33 group by `s`.`shipment_id` order by `s`.`shipment_date` desc limit 6 ;
```

Comment:
Is this something that you guys have created? It doesn't exist in the staging database.